nym-wallet/src/components/Modals/SimpleModal.tsx

```diff
@@ -31,23 +32,24 @@ export const SimpleModal: React.FC<{
   okLabel,
   sx,
   children,
+  BackdropProps,
 }) => (
-  <Modal open={open} onClose={onClose}>
+  <Modal open={open} onClose={onClose} BackdropProps={BackdropProps}>
```

Comment:
It seems like `BackdropProps` is deprecated, we should use `componentsProps.backdrop` instead -> https://mui.com/material-ui/api/modal/#props